lib/kafka/client.rb

```diff
@@ -436,15 +436,16 @@ def initialize_cluster
       )
     end
 
-    def build_ssl_context(ca_cert, client_cert, client_cert_key)
+    def build_ssl_context(ca_cert, client_cert, client_cert_key, client_cert_key_passphrase)
       return nil unless ca_cert || client_cert || client_cert_key
 
       ssl_context = OpenSSL::SSL::SSLContext.new
 
       if client_cert && client_cert_key
         ssl_context.set_params(
           cert: OpenSSL::X509::Certificate.new(client_cert),
-          key: OpenSSL::PKey.read(client_cert_key)
+          key: OpenSSL::PKey.read(client_cert_key, client_cert_key_passphrase)
```

Comment:
what happens here if you run it past a client cert key that doesn't have a passphrase?